backend/routes/v1/admins/user.router.js

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express";
+import { createUser, getUsers } from "../../../controllers/index.js"; 
+
+const router = Router();
+
+router.post("/create", createUser);
```

Comment:
remove create/get as we will be using convention like  
1) post =>{{baseUri}}/users